db/migrate/20190617191109_move_awx_credentials_to_authentications.rb

```diff
@@ -267,6 +271,15 @@ def up
     end
   rescue PG::ConnectionBad
     say("awx database is unreachable, credentials cannot be migrated")
+  rescue Fernet256::InvalidToken
+    say("awx password invalid, credentials cannot be migrated")
+    say("This is often the case when migrating a support database")
```

Comment:
This sentence is weird in upstream, since a support database isn't a thing.  Perhaps...

```
This is often the case when migrating a database that was backed up and then restored
```

Reply:
Done.